styles/codeberg/catppuccin.user.css

```diff
@@ -15,14 +15,16 @@
 @var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
 ==/UserStyle== */
 
-@-moz-document domain('codeberg.org') {
+@-moz-document domain('codeberg.org'), domain('git.auxolotl.org') {
```

Comment:
The rest looks fine, don't know why we are adding this new domain entry though?

Reply:
It was the only domain I could test on to be honest.

Reply:
This feels like a userstyle people should be adding their own custom URLs for imo.